### ISSUE TITLE
Grid showing more Items than in system

### ISSUE THREAD

**Issue by Schwarzi07** [refinedmods/refinedstorage#2936]
#### Issue description:
Grid showing more Items than in system
#### What happens:
If I take out all the Item sthe Grid still shows 310000 Items in the system which I can´t extract. 
#### Steps to reproduce:
1.Fill 32 Drives with 64K Discs
2.Put in random Item
3.Take out Items
...

#### Version (make sure you are on the latest version before reporting):

- Minecraft:1.16.5
- Forge:36.1.2
- Refined Storage:1.9.12

Does this issue occur on a server? [yes/no]
no
#### If a (crash)log is relevant for this issue, link it here:

[pastebin/gist/etc link here]

**Comment by BlueAgent** [comment]
I can't reproduce this. I tried using both 64k disks and creative Disks.

Did you make sure to spawn in new disks rather than making a copy of each disk in creative?

I tried a setup using disks copied in creative and it does result in strange behaviour like you describe (not being able to remove items but them still being duplicated and displayed).

![Setup](https://user-images.githubusercontent.com/630920/114728517-be081c80-9d71-11eb-8545-90a2f2c6f92d.png)
In order:
- 8 duped disks (reproduces the strange behaviour on a smaller scale / multiplier)
- 32 drives filled with unique creative disks (cannot reproduce)
- 32 drives filled with unique 64k disks (cannot reproduce).
